src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/NetworkTopologyConfig.java

```diff
@@ -92,6 +92,10 @@ default boolean isBfdEnabled() {
     @Default("60")
     int getPortAntiFlapStatsDumpingInterval();
 
+    @Key("switch.offline.generation.lag")
+    @Default("2")
```

Comment:
it's equal 3 in `main.yaml`, why we have different value here?

Reply:
Because I have forgotten to change it in second place (with 2 we have instability in switch lost detection - possible false positive detection).